### DISCUSSION TITLE
Is the project active?

### DISCUSSION THREAD

**Discussion by noorul** [fastapi/sqlmodel#646] [Questions]
I was evaluating sqlmodel for a project. I wonder why the last release was more than a year ago.

**Comment by 8thgencore** [comment]
Yes, it's very stressful. I think in the near future to abandon the sqlmodel and switch to SQLAlchemy

**Comment by FilipeMarch** [comment]
No, it is not active, but supposedly it's not abandoned, we hope. You can check the activity from the commit history. Nothing happened since August 2022.

The last release was more than a year ago because the project is dead since then, and we all depend 100% on `tiangolo`. Eight months ago, in January 2023, `tiangolo` [said](https://github.com/tiangolo/sqlmodel/discussions/547):
```
it [sqlmodel] will [be updated]. But there are other things I have to update do first.
```

I would say to you to not expect new features, bug fixes, or frequent updates in general in this library. 

If you are going to do a hobby project this library is great, but I would not recommend it for anyone doing some "serious business". If you do not mind depending uniquely on `tiangolo` to have any meaningful update on sqlmodel, then I believe it would be fine for you to use it.

I started using `sqlmodel` one year ago, my app has almost 40 tables at this point, and I wish I never had used sqlmodel and just used sqlalchemy from the beginning, but that's life. The lack of updates, bug fixes, new features and support in general shows this library is not well suited for a really serious project. 

I really love sqlmodel, I got used to it very quickly. The problem is that it has no active developer. Unless you want to develop this library yourself, stick to sqlalchemy, it has more features, you will be able to use pydantic V2, sqlalchemy 2.0, etc. Or wait to someday someone to do something here 😊

**Comment by noorul** [comment]
@tiangolo Any roadmap for a new release? What do you suggest? SQLAlchemy is pointing to this library for pydantic integration.

**Comment by clstaudt** [comment]
This library has more than 10k stars and 7k projects using it, and a dependency on a single developer is blocking everything and cannot be solved by the community? No matter how meritorious the developer, open source is not a one man show.

**Reply by antont** [reply]
Well, at least several folks are reporting that they are using the fork(s) mentioned there, so at least for some the community has already solved this. https://github.com/tiangolo/sqlmodel/pull/632

**Comment by 8thgencore** [comment]
It would be better if the @tiangolo  immediately said that he would no longer support the project, so that users would not torture hope.
